cilium/rockcraft.yaml

```diff
@@ -227,54 +241,36 @@ parts:
   bpftool:
     plugin: make
     source-type: git
-    source: https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git
-    source-commit: "93270357daa949e4bed375b40d0a100ce04f3399"
+    source: https://github.com/libbpf/bpftool.git
+    source-tag: v7.0.0
     source-depth: 1
-    source-subdir: "tools/bpf/bpftool"
+    source-subdir: src
+    source-submodules:
+      - "libbpf"
     build-packages:
+      - zlib1g-dev
       - libelf-dev
-    stage-packages:
-      - libelf1
-    organize:
-      usr/local/sbin/bpftool: usr/sbin/bpftool
-
-  llvm-10-bpf:
-    plugin: cmake
-    source-type: git
-    source: https://github.com/llvm/llvm-project.git
-    source-branch: "llvmorg-10.0.0"
-    source-subdir: "llvm"
-    cmake-generator: Ninja
-    override-pull: |
-      craftctl default
-      git cherry-pick 29bc5dd19407c4d7cad1c059dea26ee216ddc7ca
-      git cherry-pick 13f6c81c5d9a7a34a684363bcaad8eb7c65356fd
-      git cherry-pick ea72b0319d7b0f0c2fcf41d121afa5d031b319d5
-    override-build: |
-      cmake $CRAFT_PART_SRC_WORK -G "Ninja" \
-        -DLLVM_TARGETS_TO_BUILD="BPF" \
-        -DLLVM_ENABLE_PROJECTS="clang" \
-        -DBUILD_SHARED_LIBS="OFF" \
-        -DCMAKE_BUILD_TYPE="Release" \
-        -DLLVM_BUILD_RUNTIME="OFF" \
-        -DCMAKE_INSTALL_PREFIX="/usr/local"
-
-      ninja clang llc llvm-objcopy
-
-      cp -r $CRAFT_PART_BUILD/bin $CRAFT_PART_INSTALL
-    stage:
-      - -bin
-    organize:
-      bin/clang-10: usr/bin/clang
-      bin/llc: usr/bin/llc
-      bin/llvm-objcopy: usr/bin/llvm-objcopy
```

Comment:
We are dropping `llvm-10-bpf` part entirely. This was just statically built llvm tools with only the `bpf` target turned on. The cherry picked commits are there to enable the `bpf` targets / specific to that. Cilium also states that technically any `llvm+clang >= 10` with bpf target turned on is good, check [here for reference](https://docs.cilium.io/en/stable/operations/system_requirements/#summary). So instead of pulling the whole llvm-10 source and building it, we pull it and stage it from the apt repository.